index.js

```diff
@@ -20,7 +20,7 @@
  * @callback Visitor
  * @param {V} node Found node
  * @param {number|null} index Position of `node` in `parent`
- * @param {Parent|null} parent Parent of `node`
+ * @param {import('unist').Parent<V>|null} parent Parent of `node`
```

Comment:
This isn't the case though.

For example if you have an AST like https://astexplorer.net/#/gist/07590330c6b74c87dc6ca4c170f71bf0/c6107a73d37212d1ea8b1b6cd18cd1cef230527d

And have a visitor going to `emphasis` nodes

```ts
visit(ast, 'emphasis', (node, position, parent) => {/* do something */})
```

It would be incorrect to say `Parent<Emphasis>`, the `Parent` also contains `Text`, `Strong`, and `InlineCode`